plugins/logitech-bulkcontroller/fu-logitech-bulkcontroller-common.c

```diff
@@ -85,16 +96,27 @@ fu_logitech_bulkcontroller_proto_manager_generate_set_device_time_request(GError
 	Logi__Device__Proto__Request request_msg = LOGI__DEVICE__PROTO__REQUEST__INIT;
 
 	/* the device expects an olson_location, not a timezone */
-	olson_location = fu_common_get_olson_timezone_id(error);
-	if (olson_location == NULL)
-		return NULL;
+	if (fu_device_has_flag(device, FWUPD_DEVICE_FLAG_EMULATED) ||
+	    fu_device_has_flag(device, FWUPD_DEVICE_FLAG_EMULATION_TAG)) {
+		olson_location = g_strdup("Europe/London");
+	} else {
+		olson_location = fu_common_get_olson_timezone_id(error);
+		if (olson_location == NULL)
+			return NULL;
+	}
```

Comment:
Part of me wants to see this in the common layer, but maybe that can wait until next plugin that needs the info.